common/common.go

```diff
@@ -105,6 +141,16 @@ func LogError(logType LogType, v ...interface{}) {
 	os.Exit(1)
 }
 
+// LogErrorNoExit internal, used in all packages
```

Comment:
This comment does not describe what function does.
That doc comment should say something about logging and string message return.

[Documenting go code](https://blog.golang.org/godoc-documenting-go-code).

Example:
```
LogErrorNoExit writes provided values to default logger if logType matches current log type.
Returns message that was printed.
```

Note: I know that common.go contains more such oddly documented functions that are out-of-scope for this review.

Reply:
If desired, "internal" package can be achieved since Go 1.5 with special import path:
```
An import of a path containing the element “internal” is disallowed if the importing code is outside the tree rooted at the parent of the “internal” directory.
```
This may require slight project layout re-organization, but it's more reliable way to make something "internal".

Reply:
What do you think about this:
1. rename `LogError` to `LogFatal`
2. rename `LogErrorNoExit` to `LogError`
?

Reply:
Yes, I think it should be refactored separately, this pull request is already too large.